e2e/favorites.spec.ts

```diff
@@ -1,13 +1,120 @@
-import { expect, test } from '@playwright/test'
+import { Page, expect, test } from '@playwright/test'
 import { testAuth } from './fixtures/authenticatedUserPages'
+import { entityUrlPathname } from './helpers/entities'
 import { goToDashboard } from './helpers/testUser'
+import { Project } from './helpers/types'
+import { waitForInitialPageLoad } from './helpers/utils'
+
+const PUBLIC_PROJECT: Project = {
+  name: 'swc-e2e-test-project',
+  id: 'syn12366772',
+}
```

Comment:
For now, I set up a public project owned by a swc-e2e-admin. In the future, we could create/delete a new public project within the test instead.